### ISSUE TITLE
Key down LED

### ISSUE THREAD

**Issue by g7kse** [nealey/vail-adapter#4]
Hi Neale,

Just ordering some parts for this and will be putting this in a small enclosure from [Hammond](https://www.hammfg.com/files/parts/pdf/1455C801RD.pdf) that I tend to use a bit so I'll adapt the PCB that's in the repo to slide in. Just wondering if adding a key down (TX'ing) LED is possible. The enclosure is 'desktop' so less like the smaller ones and there's obviously loads of space on the board

I would have thought using the buzzer might be a solution without changing code but then I noticed that Pin 10 might be used instead of Pin 9. 

Before i go and fab up some PCB's can you let me know if the pin allocations will change and if its practical to take an LED off the buzzer pin?

**Comment by nealey** [comment]
Yes, there's a bug with the current PCB, thanks for reminding me. I just opened #4 to track this problem. If you'd like to submit a patch to the PCB files, that would be super great! I'm not very adept at the software that generated them: I don't even remember what it's called.

You should be able to hook an LED up instead of a speaker: I think it uses the same PWM code, so the LED will not be full brightness. I don't have anything nearby to test this out, but if you have a microcontroller, go ahead and hook a sacrificial LED up to it and report back on how it went! The LED should be fine, but you never know, so don't use your favorite LED ;)

A very small code change would make a TX LED easy on another pin. You could `pinMode(8, OUTPUT)` and then, in beginTx and endTx, add `digitalWrite(8, true)` and `digitalWrite(8, false)` respectively, to get that LED going. I'd be glad to accept a patch adding this in, as well: but please use a `#define` instead of a hard-coded pin number :)

**Comment by nealey** [comment]
@g7kse I finally got around to investigating: it appears the PCB files checked in have the speaker wired up to the correct pin. The board has 2023-01-22 silkscreened onto it: the ones I ordered are 2023-01-21. I must have realized shortly after I submitted the order that the pin was wrong, fixed it in git, and not bothered ordering a new batch, because I broke out every pin on top of the board anyway.

Anyway, I am strongly opposed to changing pins in the code, so feel encouraged to rely on the current allocations!

I'm going to close this issue, but I'd love to hear if you had any success wiring an LED to the buzzer pin.
